closure/compiler/closure_js_binary.bzl

```diff
@@ -67,12 +68,20 @@ def _impl(ctx):
     args += ["--debug"]
   elif is_using_closure_library(srcs):
     args += ["--define=goog.DEBUG=false"]
+  if is_using_closure_library(srcs) and language_out in ["ECMASCRIPT6_TYPED",
+                                                         "ECMASCRIPT6_STRICT",
+                                                         "ECMASCRIPT5_STRICT"]:
+    args += ["--define=goog.STRICT_MODE_COMPATIBLE"]
+  if contains_file(srcs, ctx.attr._soyutils_usegoog):
```

Comment:
Good point.

As the filename changes with the version (eg: `soy-2016-01-12-soyutils_usegoog.js`) I was attempting to abstract it via the label. I'm fixing the bug in the next patch set (still via the label though). Please let me know if you prefer a different solution. I'm happy to make another revision.